### PR TITLE
Reaction to SplitPackages change where it does not produce an executa…

### PR DESCRIPTION
**Pull request by kichalla**
…ble anymore

### CODE REVIEW

File: build/repo.targets [hunk 1]
```diff
@@ -40,18 +40,18 @@
...
 
-  <Target Name="RunSplitPackages" DependsOnTargets="RunResolvePackageDependencies">
+  <Target Name="RunSplitPackages" DependsOnTargets="RunResolvePackageDependencies;_FindDotNetPath">
     <ItemGroup>
       <SplitPackagePath Include="%(PackageDefinitions.ResolvedPath)" Condition="'%(PackageDefinitions.Name)'=='SplitPackages'" />
     </ItemGroup>
     <PropertyGroup>
       <ProductPackagesSource>$(ArtifactsDir)product-packages</ProductPackagesSource>
       <SplitPackagesCsv>$(RepositoryRoot)packages\packages.csv</SplitPackagesCsv>
-      <SplitPackagesBinary>@(SplitPackagePath)\SplitPackages.exe</SplitPackagesBinary>
```
Comment (kichalla) [inline build/repo.targets:50]: I haven't checked the SplitPackage change in yet, but just wanted to get this PR out first

Comment (pranavkm) [reply build/repo.targets:50]: :shipit:  as long as it works

**Comment by kichalla** [comment]
@natemcmaster Just FYI since we talked earlier about converting SplitPackages into task. Actually I am not converting it.

**Review by pranavkm — APPROVED** [review]

File: build/repo.targets [hunk 2]
```diff
@@ -40,18 +40,18 @@
...
+      <SplitPackagesBinary>@(SplitPackagePath)\SplitPackages.dll</SplitPackagesBinary>
       <SplitPackagesArgs>--source $(ProductPackagesSource) --csv $(SplitPackagesCsv) --destination $(ArtifactsDir)</SplitPackagesArgs>
     </PropertyGroup>
 
-    <Exec Command="$(SplitPackagesBinary) $(SplitPackagesArgs)" />
+    <Exec Command="$(DotNetPath) $(SplitPackagesBinary) $(SplitPackagesArgs)" />
```
Comment (natemcmaster) [inline build/repo.targets]: `&quot;$(DotNetPath)&quot; &quot;$(SplitPackagesBinary)&quot;`

Comment (natemcmaster) [reply build/repo.targets]: In case path has spaces in it.

File: build/repo.targets [hunk 3]
```diff
@@ -40,18 +40,18 @@
...
-  <Target Name="RunSplitPackages" DependsOnTargets="RunResolvePackageDependencies">
+  <Target Name="RunSplitPackages" DependsOnTargets="RunResolvePackageDependencies;_FindDotNetPath">
     <ItemGroup>
       <SplitPackagePath Include="%(PackageDefinitions.ResolvedPath)" Condition="'%(PackageDefinitions.Name)'=='SplitPackages'" />
     </ItemGroup>
     <PropertyGroup>
       <ProductPackagesSource>$(ArtifactsDir)product-packages</ProductPackagesSource>
       <SplitPackagesCsv>$(RepositoryRoot)packages\packages.csv</SplitPackagesCsv>
-      <SplitPackagesBinary>@(SplitPackagePath)\SplitPackages.exe</SplitPackagesBinary>
+      <SplitPackagesBinary>@(SplitPackagePath)\SplitPackages.dll</SplitPackagesBinary>
```
Comment (natemcmaster) [inline build/repo.targets:50]: Can be simplified using [Item functions](https://msdn.microsoft.com/en-us/library/ee886422.aspx)
```xml
<SplitPackagesBinary>@(PackageDefinitions->WithMetadataValue('Name','SplitPackages')->Metadata('ResolvedPath'))\SplitPackages.dll</SplitPackagesBinary>
```

Comment (kichalla) [reply build/repo.targets:50]: Ok. I didn't notice your comment, but will do it in a follow up commit

**Pull request merged**